src/lib/fauna-import-writer.js

```diff
@@ -44,20 +46,47 @@ function getFaunaImportWriter(
     }
   }
 
+  /**
+  Status codes of interest:
+  * [409] Contention - attempt to retry
+  * [410] Account disabled - do not retry
+  * [413] Request too large - should not happen
+  * [429] Too many requests - attempt to retry
+  * [503] Timeout - do not retry
+  */
+  const retryHandler = (e) => {
+    switch (e.requestResult?.statusCode) {
+      case 409:
+        return true
+      case 429:
+        return true
+      default:
+        return false
+    }
+  }
+
   const requestBatch = (batch) => {
     // TODO have the call (or client) return the write-ops
-    return client.query(
-      q.Do(
-        batch.map((data) =>
-          q.Create(q.Collection(collection), {
-            data: Object.keys(data).reduce(
-              (memo, next) => ({ ...memo, [next.trim()]: data[next] }),
-              {}
-            ),
-          })
+    const write = (batch) =>
+      client.query(
+        q.Do(
+          batch.map((data) =>
+            q.Create(q.Collection(collection), {
+              data: Object.keys(data).reduce(
+                (memo, next) => ({ ...memo, [next.trim()]: data[next] }),
+                {}
+              ),
+            })
+          )
         )
       )
-    )
+    if (allowRetries)
+      return backOff(() => write(batch), {
```

Comment:
what is the backoff algorithm?

Reply:
i think you need to be more particular about configuring the backoff algorithm:

https://github.com/coveooss/exponential-backoff#ibackoffoptions

Reply:
typically you'll see 500 ms - 1s for an initial backoff (the initial delay); and usage of 2 as an expontent for growing it (the time multiple).